jetbrains-core/src/software/aws/toolkits/jetbrains/core/explorer/nodes/AwsExplorerRootNode.kt

```diff
@@ -16,8 +16,9 @@ import software.aws.toolkits.jetbrains.core.region.AwsRegionProvider
 class AwsExplorerRootNode(private val nodeProject: Project) : AbstractTreeNode<Any>(nodeProject, Object()) {
     private val regionProvider = AwsRegionProvider.getInstance()
     private val settings = ProjectAccountSettingsManager.getInstance(nodeProject)
+    private val EP_NAME = ExtensionPointName<AwsExplorerServiceNode>("aws.toolkit.explorer.node")
 
-    override fun getChildren(): List<AwsExplorerNode<*>> = AwsExplorerService.values()
+    override fun getChildren(): List<AwsExplorerNode<*>> = EP_NAME.extensions
```

Comment:
nit: we use `getExtensionList` elsewhere which returns an actual List vs an array